Automattic/wp-calypso
# Analytics: enable/disable ad-trackers via config

This PR allows ad-trackers to be  easily enabled and disabled via config file. When an ad-tracker is disabled it will not fire any event and its relative script will not be loaded at all.

Note that when "ad-tracking" is disabled in the config file then all "ad-tracking/*" entries are ignored.

This PR contextually disables the following ad-trackers:
- Media Wallah
- Quora
- Pandora
- Atlas

# Testing
- in `config/development.json` enable `"ad-tracking": true`
- restart Calypso
- open http://calypso.localhost:3000
- open the Network tab
- reload the page
- the following entries should be present: fbevents.js, conversion_async.js, bat.js, ld.js, ytc.js, uwt.js, 6355556.fls.doubleclick.net, insight.min.js
- the following entries should not be present: mwData.min.js, qevents.js

Example:

![2017-09-06 12 38 47 f4rrz](https://user-images.githubusercontent.com/10284338/30110032-e936b4ba-9308-11e7-9af8-db251bd10658.png)

![2017-09-06 12 39 33 i79pq](https://user-images.githubusercontent.com/10284338/30110039-ec844b0a-9308-11e7-9464-5ec27721aaf5.png)
